fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBExceptions.java

```diff
@@ -130,6 +131,13 @@ public FDBStoreRetriableException(FDBException cause) {
     }
 
     public static RuntimeException wrapException(@Nonnull Throwable ex) {
+        //transfer any logging details into mapped exception
+        Object[] logInfo;
+        if (ex instanceof LoggableKeysAndValues) {
+            logInfo = ((LoggableKeysAndValues)ex).exportLogInfo();
+        } else {
+            logInfo = new Object[0];
```

Comment:
Minor, but it would be good to save the unnecessary object allocation here, by defining a:

    private static final Object[] EMPTY_KEYS_AND_VALUES = new Object[0];

at the top of the class, then doing:

    logInfo = EMPTY_KES_AND_VALUES;